recipes-browser/wpewebkit/wpewebkit.inc

```diff
@@ -7,13 +7,14 @@ LICENSE = "BSD & LGPLv2+"
 LIC_FILES_CHKSUM = "file://Source/WebCore/LICENSE-LGPL-2.1;md5=a778a33ef338abbaf8b8a7c36b6eec80 "
 
 DEPENDS = " \
-    bison-native gperf-native harfbuzz-native libxml2-native ccache-native ninja-native ruby-native cairo \
```

Comment:
AFAIK libxml2-native is only need in buildtime and from the host.

Reply:
Can you explain glib-2.0 -> glib-2.0-native. AFAIK that removes the dependency for target arch what I think is not desiderable.

Reply:
> I miss ccache-native here. This is useful if you set `INHERIT += "ccache"` in the local.conf.

For above you don't need to depends on ccache-native as this is done magically by yocto for you.
FYI, wpewebkit is not compatible with the`INHERIT += "ccache"` and if you gonna use it you would have to disable:

>CCACHE_DISABLE[unexport] = "1"

and also disable using ccache by webkit. This is because INHERIT += "ccache" already invokes the compile via ccache (so in wpewebkit you would end up in something like `ccache ccache g++` - which wouldn't work for you.

Reply:
> Can you explain glib-2.0 -> glib-2.0-native. AFAIK that removes the dependency for target arch what I think is not desiderable.

Without glib-2.0-native it doesn't compile because of:
>glib-compile-resources: command not found
>glib-mkenums: command not found